### ISSUE TITLE
Attempting to upgrade aws-lambda-go version in a project with go.mod fails

### ISSUE THREAD

**Issue by alex** [aws/aws-lambda-go#222]
```
alex@penguin /t/x> cat go.mod 
module foo

go 1.12

require github.com/aws/aws-lambda-go v1.8.1
alex@penguin /t/x> go get -u github.com/aws/aws-lambda-go
go: gopkg.in/urfave/cli.v1@v1.21.0: go.mod has non-....v1 module path "github.com/urfave/cli" at revision v1.21.0
go get: error loading module requirements
```

**Comment by michael-i2o** [comment]
I've just hit this issue. @alex thanks for creating a PR so quickly!

**Comment by ckeyes88** [comment]
As a work around until #223 gets resolved you can use the Go module `replace` directive to specify the correct package...

`gopkg.in/urfave/cli.v1 => github.com/urfave/cli v1.21.0`

**Comment by bmoffatt** [comment]
released @223 with v1.12.1
